src/lib.rs

```diff
@@ -1257,7 +1315,7 @@ mod tests {
     fn load_symbolfilecfg_process() {
         // Check if SymbolSrcCfg::Process expands to ELFResolvers.
         let cfg = vec![SymbolSrcCfg::Process { pid: None }];
-        let cache_holder = CacheHolder::new();
+        let cache_holder = CacheHolder::new(true);
```

Comment:
we'll probably have a bunch of such options and having them as bool arguments will get confusing very fast. I think having some sort of struct with named fields is better longer-term. Please consider following up with such API change, thanks!

Reply:
A new CacheHolderOpts type is added.